modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/OrthologyStats.pm

```diff
@@ -83,12 +96,14 @@ sub fetch_input {
         $identity, $hom_identity) = ($row->{homology_type}, $row->{gene_tree_node_id}, $row->{gene_member_id}, $row->{hom_gene_member_id}, 
         $row->{genome_db_id}, $row->{hom_genome_db_id}, $row->{identity}, $row->{hom_identity});
         
+        ( $genome_db_id, $hom_genome_db_id ) = ( $gdb_map->{$genome_db_id}, $gdb_map->{$hom_genome_db_id} );
+        
         $stats1{$homology_type}->{$gene_tree_node_id}->{$gene_member_id}->{$genome_db_id}->{"num_homologies"} += 1; # n1
         $stats1{$homology_type}->{$gene_tree_node_id}->{$hom_gene_member_id}->{$hom_genome_db_id}->{"num_homologies"} += 1; # n2
         $stats1{$homology_type}->{$gene_tree_node_id}->{$gene_member_id}->{$genome_db_id}->{"sum_perc_id"} += $identity; # p1
         $stats1{$homology_type}->{$gene_tree_node_id}->{$hom_gene_member_id}->{$hom_genome_db_id}->{"sum_perc_id"} += $hom_identity; # p2
     }
-    close $hom_handle;   
+    close $hom_handle;
```

Comment:
```suggestion
    close $hom_handle;   
```
(leaving it as it was)

Reply:
Once that's changed, @CristiGuijarro , can you squash all your commits into one ?